test/tstDetailsKokkosExtUninitializedMemoryAlgorithms.cpp

```diff
@@ -0,0 +1,61 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include "ArborX_EnableDeviceTypes.hpp" // ARBORX_DEVICE_TYPES
+#include <ArborX_DetailsKokkosExtUninitializedMemoryAlgorithms.hpp>
+#include <ArborX_Exception.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include "BoostTest_CUDA_clang_workarounds.hpp"
+#include <boost/test/unit_test.hpp>
+
+#define BOOST_TEST_MODULE KokkosExtUninitializedMemoryAlgorithms
```

Comment:
Sorry for noticing late but what is the intent here?  I don't think that does what we think assuming the intent was to resister a test suite with all unit tests in that source file.

Reply:
What do you mean? I'm not 100% familiar with what it does, and just followed the way it's done in other KokkosExt testing files. From the [documentation](https://beta.boost.org/doc/libs/1_82_0/libs/test/doc/html/boost_test/usage_variants.html) it seems that it only needs to be done in one translation module and before inclusion of `unit_test.hpp` header.